travelpal/assets/js/components/Profile.jsx

```diff
@@ -4,10 +4,66 @@ import PropTypes from 'prop-types';
 import api from '../api';
 import ProfileInfo from './ProfileInfo';
 import ProfileFriends from './ProfileFriends';
+import { FormGroup, Label, Input, Button } from 'reactstrap';
+import SearchResults from './SearchResults';
 
 // Renders the user's profile
-export default function Profile({ userInfo, friends, formOnChange,
-  submitOnClick }) {
+export default function Profile({
+  userInfo,
+  friends,
+  formOnChange,
+  submitOnClick,
+  userId,
+  users,
+  search,
+  updateSearch
+}) {
+  // Updates the form with the query
+  function update(ev) {
+    const query = $(ev.target).val().toLowerCase();
+    updateSearch(query);
+  }
+
+  function toggleDefault() {
```

Comment:
You don't need two separate functions. You can just use one function that calls `.toggle()` on everything, which will switch all shown divs to hidden and vice versa.